app/api/views/api_view.py

```diff
@@ -0,0 +1,21 @@
+from app.serializers.player_serializer import PlayerSerializer
+from app.models import Player, Spectator, Moderator
+
+from django.http import Http404, JsonResponse
+
+import json
+
+def account_info(request):
+    if not request.user.is_authenticated:
+        print("Not Authenticated")
+        raise Http404 
+
+    if request.method == "GET":
+        playerSet = Player.objects.all().filter(user=request.user)
+        if playerSet:
+            player = playerSet[0]
+            serializer = PlayerSerializer(player)
+            return JsonResponse(serializer.data, safe=False)
+        else:
+            print("Not found")
```

Comment:
I'm not sure if we should be printing here, or if we are, it should give a lot more context as to what was not found and what the issue is. The same holds for above.